docs/src/tutorials/randomized.md

```diff
@@ -0,0 +1,944 @@
+# Tutorial on Checking ERC20 with Property-Based Testing and TLA+
```

Comment:
The name of this file doesn't seem to clearly relate to the content. Would it be possible to use a more related name? Maybe `pbt-and-tla` or something?

Reply:
Good point. Just renamed.